applications/luci-app-statistics/root/etc/config/luci_statistics

```diff
@@ -191,3 +191,6 @@ config statistics 'collectd_thermal'
 config statistics 'collectd_uptime'
 	option enable '0'
 
+config statistics 'collectd_dhcpleases'
```

Comment:
The config section should be added to the correct alphabetical place among the input plugins

Reply:
changed :)